frontend/src/metabase/containers/dnd/ItemsDragLayer.jsx

```diff
@@ -64,7 +64,7 @@ class DraggedItems extends Component {
     // necessary for decent drag performance
     return (
       nextProps.items.length !== this.props.items.length ||
-      nextProps.pinnedItems.length !== this.props.pinnedItems ||
+      nextProps.pinnedItems.length !== this.props.pinnedItems.length ||
```

Comment:
@sloansparger Just wanted to flag this new substantive change. Somehow there was a new dragging performance issue introduced in this branch, and this fixes it